### ISSUE TITLE
ImageNet-1K

### ISSUE THREAD

**Issue by cile98** [yongchao97/FRePo#8]
Could you please provide the hyperparameters used for training the model on ImageNet synthetic data? (eg learning rate, number of epochs, flip, ZCA and etc)?

Many thanks for considering my request.

**Comment by yongchao97** [comment]
Hi,

Thanks for reaching out. I use the default hyperparameter most of the case. Specifically, for ImageNet I use an Image resolution of 64x64 with ZCA normalization, learning rate = 3e-4, number of steps = 2M, flip = False.

Best,
Yongchao
